src/mrinufft/trajectories/densities.py

```diff
@@ -0,0 +1,134 @@
+"""Sampling densities and methods."""
+
+import numpy as np
+import numpy.fft as nf
+import numpy.linalg as nl
+import numpy.random as nr
```

Comment:
I think this sacrifices a bit on readability. I would rather prefer np.random.<> np.linalg.<> etc..

Reply:
It is true that those aliases are only used a couple of times, there is no need for them